src/ui/public/agg_types/metrics/bucket_avg.js

```diff
@@ -13,6 +14,17 @@ export function AggTypesMetricsBucketAvgProvider(Private) {
     subtype: siblingPipelineHelper.subtype,
     params: [
       ...siblingPipelineHelper.params()
-    ]
+    ],
+    getValue: function (agg, bucket) {
+      const customMetric = agg.params.customMetric;
+      const scalingMetric = customMetric.type && (customMetric.type.name === 'count' || customMetric.type.name === 'sum');
```

Comment:
I created an "isScalable" function on the metric type and count and sum return true.